src/data/scout_data.py

```diff
@@ -107,40 +111,6 @@ def filter_records(self, field, value_list, mask=False, exclusion_analysis=False
         """
         data = self.data
         self.data = utility.filter_records(data, field, value_list, self.logger, mask, exclusion_analysis)
-        self.min_year, self.max_year = utility.years_of_return(self.data["Year"])
-
-    def nearby_records(self, field, value_list, distance):
```

Comment:
Apologies think I'm missing the plot here, why can this function be deleted?

Reply:
Ah I think I got it, from the functions in geography.py?

Reply:
Yeah the function was only used once so I removed a layer of abstraction - here 432e813